### DISCUSSION TITLE
How to use native image lazy loading inside swiper?

### DISCUSSION THREAD

**Discussion by RWave88** [nolimits4web/swiper#5580] [Q&A]
I would like to use native image lazy loading inside swiper slides. Is this actually possible? 
Why I don't want to use custom solution: I use swiper.js inside Nuxt, so I need src attribute to stay consistent between client/server, moreover, usage of native lazy loading feels much more relevant to me in a long run

**Comment by JohnMica** [comment]
are you trying to use lazy images inside each slide ?
if so, you could simply bind the image src to the index / name of the slider content ?
natively the loading='lazy' should only load the image when in observable view

```
 <img
      height="40"
      loading="lazy"
      :src="`${index}/image_path/${index}`"
  />
```
